### PR TITLE
added style to page

### PR DESCRIPTION
**Pull request by aeoneis**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,36 +2,27 @@
 <html>
   <head>
     <title>Bus Mall Products</title>
+    <link href="https://fonts.googleapis.com/css?family=Lobster+Two|Poiret+One" rel="stylesheet">
```
Comment (alanafrankly) [inline index.html]: Screencap, please!

File: js/app.js [hunk 2]
```diff
@@ -87,46 +95,62 @@ function setup() {
...
   }
 }
 
+// to render image object
 function render(productObj) {
   var image = document.createElement('img');
   image.setAttribute('src', 'images/' + productObj.path);
   image.setAttribute('id', productObj.description);
+  image.setAttribute('class', 'productImage');
   image.setAttribute('width', '300px');
```
Comment (alanafrankly) [inline js/app.js]: Add css class that gives these styles to img.productImage

File: js/app.js [hunk 3]
```diff
@@ -195,11 +220,6 @@ function chart() {
     dataArray[1].push(products[i].clicked);
   }
 
-  // displayedProducts.forEach(function(product){
-  //   data.push(product.clicked);
-  // });
-
-  console.log('test data array' + dataArray);
   // modeled after the Getting Started example in the chartJS documentation
   var chart = new Chart(ctx, {
```
Comment (alanafrankly) [inline js/app.js]: Get rid of var chart, should work with just new Chart(...

File: js/app.js [hunk 4]
```diff
@@ -246,10 +270,12 @@ function createOrUpdateProductState(leftProduct, centerProduct, rightProduct){
   return parsedProductState;
 }
 
+// delete from local storage
 function deleteProductState () {
   localStorage.removeItem('productState');
 }
 
+// clear all from local storage
```
Comment (alanafrankly) [inline js/app.js]: Good addition of comments throughout.

File: index.html (hunk 1 above)
Comment (aeoneis) [inline index.html]: busmallproducts

File: js/app.js (hunk 3 above)
Comment (aeoneis) [inline js/app.js]: Thank you!

**Pull request merged**